neo/Consensus/ConsensusService.cs

```diff
@@ -109,7 +109,7 @@ private void CheckCommits()
 
         private void CheckExpectedView(byte viewNumber)
         {
-            if (context.ViewNumber == viewNumber) return;
+            if (context.ViewNumber >= viewNumber) return;
             // if there are `M` change view payloads with NewViewNumber greater than viewNumber, then, it is safe to move
             if (context.ChangeViewPayloads.Count(p => p != null && p.GetDeserializedMessage<ChangeView>().NewViewNumber >= viewNumber) >= context.M())
```

Comment:
Maybe this line 114 would safeguard this change to a lower view. But I am not sure.
Anyway, this fix was surely necessary. Nice catch, Erik.